lib/tfa/cli.rb

```diff
@@ -26,5 +27,13 @@ def totp(name = nil)
     def storage
       @storage ||= Storage.new(filename: options[:filename] || 'tfa')
     end
+
+    def clean(secret)
+      if secret.include?('=')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.